vocabularies/Common.xml

```diff
@@ -232,15 +232,12 @@
       </Term>
 
       <Term Name="SecondaryKey" AppliesTo="EntityType" Type="Collection(Edm.PropertyPath)" Nullable="false">
-        <Annotation Term="Core.Revisions">
-          <Collection>
-            <Record>
-              <PropertyValue Property="Kind" EnumMember="Core.RevisionKind/Deprecated" />
-              <PropertyValue Property="Description" String="Use term `AlternateKeys` from the OASIS Core vocabulary instead" />
-            </Record>
-          </Collection>
+        <Annotation Term="Core.Description" String="The listed properties form a secondary key" />
+        <Annotation Term="Core.LongDescription">
+          <String>Multiple secondary keys are possible using different qualifiers.
+          Unlike [`Core.AlternateKeys`](https://github.com/oasis-tcs/odata-vocabularies/blob/main/vocabularies/Org.OData.Core.V1.md#AlternateKeys),
+          secondary keys need not support addressing an entity in a resource path.</String>
```

Comment:
What shall the client make of this? How does it know whether it can use a secondary key in a resource path? That's why I wrote "cannot".
```suggestion
          Unless also annotated as [`Core.AlternateKeys`](https://github.com/oasis-tcs/odata-vocabularies/blob/main/vocabularies/Org.OData.Core.V1.md#AlternateKeys),
          secondary keys cannot be used to to address an entity in a resource path.</String>
```

Reply:
The client cannot know, and we can't either. Maybe the service supports secondary keys in resource paths tomorrow.